lib/toml-rb/dumper.rb

```diff
@@ -5,7 +5,7 @@ class Dumper
     attr_reader :toml_str
 
     def initialize(hash)
-      @toml_str = ""
+      @toml_str = +""
```

Comment:
Rather than making it mutable, would it be better to avoid mutating in the first place? So change `<<` to `+=` here:
https://github.com/emancu/toml-rb/blob/04898eb6428d49d14f6b31234a4c4f2b0ac4eabe/lib/toml-rb/dumper.rb#L56
It would be consistent with the other modification:
https://github.com/emancu/toml-rb/blob/04898eb6428d49d14f6b31234a4c4f2b0ac4eabe/lib/toml-rb/dumper.rb#L86

Reply:
Nice one @ekohl !

@bastelfreak would you mind updating the PR to include this change?